### ISSUE TITLE
bel2rdf: prefixes missing

### ISSUE THREAD

**Issue by mhsano** [OpenBEL/bel.rb#71]
In only a very small number of cases bel2rdf (v0.3.3) removes the prefix;  e.g. an example converting the large_corpus.bel file into RDF  for FOXO1 :  

transcriptionalActivity(proteinAbundance(HGNC:FOXO1)) increases rnaAbundance(HGNC:ST6GALNAC2)

is converted into:

bel:tscript_p_FOXO1_Increases_r_HGNC_ST6GALNAC2 belv:hasSubject bels:tscript_p_FOXO1

while it should be something like:

bel:tscript_p_HGNC_FOXO1_Increases_r_HGNC_ST6GALNAC2        belv:hasSubject bel:tscript_p_HGNC_FOXO1

In these cases bel2rdf also does not produce hasConcept statements.

**Comment by abargnesi** [comment]
@mhsano Thanks for this report!

I have been able to confirm this on the large corpus for `HGNC:FST` abundances using version *0.3.3*. An example of a statement in the [20150611 version of the large corpus](http://resource.belframework.org/belframework/20150611/knowledge/large_corpus.bel) is on, line 8550:

```
p(HGNC:IL10) increases complex(p(HGNC:FST),p(HGNC:INHBA),p(HGNC:INHBA))
```

The RDF translation includes the following ntriples for the `p(HGNC:FST)` term:

```
<http://www.openbel.org/bel/p_FST> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.openbel.org/vocabulary/Term> .
<http://www.openbel.org/bel/p_FST> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.openbel.org/vocabulary/ProteinAbundance> .
<http://www.openbel.org/bel/p_FST> <http://www.w3.org/2000/01/rdf-schema#label> "p(FST)" .
```

This test was run two ways using the [20150611 version of the large corpus](http://resource.belframework.org/belframework/20150611/knowledge/large_corpus.bel):

1. **bel command**
  - `bel translate -i large_corpus.bel bel rdf > large_corpus.n3`

2. **bel2rdf.rb command**
  - `bel2rdf.rb -b large_corpus.bel -f turtle > large_corpus.ttl`


I will debug further this week.

**Comment by abargnesi** [comment]
Closed via #99. Will be released in 0.5.0.
